core/metrics/cgroups/v1/memory.go

```diff
@@ -591,6 +600,9 @@ var memoryMetrics = []*metric{
 		unit: metrics.Bytes,
 		vt:   prometheus.GaugeValue,
 		getValues: func(stats *v1.Metrics) []value {
+			if stats.Memory == nil {
```

Comment:
Don't `GetMemory()` and chained funcs already handle this?

Reply:
Yep,  I can close that.

But should not we keep the check nil before ?  this will reduce the consule of method call stack twice.